broskichat-app/src/App.css

```diff
@@ -25,4 +25,13 @@
 @keyframes App-logo-spin {
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
+} */
+
+/* Custom CSS */
+.App {
+  margin: 300px;
+}
+
+.footerBG {
+  background-color: #333;
```

Comment:
I see a theme forming! Once the theme fills out a little more, we'll want to override the bulma sass variables. This is fine for now though 👍